gpkit/solution_array.py

```diff
@@ -112,12 +112,12 @@ def table(self, tables=("cost", "sweepvariables", "freevariables",
                     continue
                 strs += ["\n%s\n----" % table_title]
                 if len(self) > 1:
-                    costs = ["%-8.3g" % c for c in subdict[:4]]
+                    costs = ["%-8.3g" % c for c in mag(subdict[:4])]
                     strs += [" [ %s %s ]" % ("  ".join(costs),
                                              "..." if len(self) > 4 else "")]
                     cost_units = self.program[0].cost.units
                 else:
-                    strs += [" %-.4g" % subdict]
+                    strs += [" %-.4g" % mag(subdict)]
```

Comment:
I haven't yet gotten pint's units to look the way I want, and this lets us be consistent with nomial units printing.